kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/players/BaseAudioPlayer.kt

```diff
@@ -174,8 +174,8 @@ abstract class BaseAudioPlayer internal constructor(private val context: Context
      * Stops playback, resetting the player and queue.
      */
     open fun stop() {
-        exoPlayer.stop(true)
-        exoPlayer.pause()
+        exoPlayer.stop()
+        exoPlayer.clearMediaItems()
```

Comment:
New ExoPlayer versions have some deprecated functions we used to use, and those were replaced with the proper ones above.